Source/JavaScriptCore/builtins/JSIteratorPrototype.js

```diff
@@ -256,12 +282,15 @@ function find(predicate)
     if (!@isObject(this))
         @throwTypeError("Iterator.prototype.find requires that |this| be an Object.");
 
-    if (!@isCallable(predicate))
+    var iterated = this;
```

Comment:
Let's just use `this` directly in this function instead of `iterated` when possible.